README.rst

```diff
@@ -4,39 +4,17 @@ MongoDB Java Driver Documentation
 
 This repository contains documentation for the MongoDB Java Driver.
 
-Build and Stage
----------------
-
-To build and stage the documentation, use the
-`snooty autobuilder <https://docs.google.com/document/d/1tAr5kZL0xWhAbfmn9AVsIXbF9quaFFK9nfbUhpKJEBU/edit#heading=h.lwy5m08dmc0i>`_.
-To get the snooty autobuilder running on your fork, add a webhook
-according to the autobuilder setup guide. Feel free to reach out
-on the *snooty-onboarding* Slack channel if you cannot access
-the `how-to guide <https://docs.google.com/document/d/1x6NZExTldbAZCOMdKUVdPeI-ukMM1S6qZhbKjXAZ3uQ/edit>`_.
-
-Contribute
-----------
-
-To contribute to the documentation,
-
-- If you have not done so already, please sign the `MongoDB Contributor
-  Agreement <https://www.mongodb.com/legal/contributor-agreement>`_.
-
-- Fork this repository on GitHub and issue a pull request.
-
-See the following documents that provide an overview of the
-documentation style and process:
-
-- `Style Guide <https://mongodb.org/docs/meta/style-guide/style/>`_
-- `Documentation Organization <https://mongodb.org/docs/meta/organization>`_
-- `Documentation Practices <https://mongodb.org/docs/meta/practices>`_
 
 File JIRA Tickets
 -----------------
 
-File issue reports or requests at the `Documentation Jira Project
+Please file issue reports or requests at the `Documentation Jira Project
```

Comment:
Issues:
1) The "Contribute" section duplicates this information and offers a title associated with the reason rather than the recommended action ("File JIRA Tickets").
2) "at" doesn't seem like the correct preposition to "file" reports/requests.
3) "Jira" -> "JIRA"

Suggestion: Remove this section and put corrected text under the "Contribute" heading: "Please report issues or requests in the Documentation JIRA project..."